firstvoices/backend/views/search/utils.py

```diff
@@ -0,0 +1,45 @@
+from backend.models.dictionary import DictionaryEntry
+from backend.search_indices.dictionary_entry_document import (
+    ELASTICSEARCH_DICTIONARY_ENTRY_INDEX,
+)
+
+
+def hydrate_objects(raw_objects):
+    """
+    Adding required properties to raw objects returned form elastic-search.
+    """
+    complete_objects = []
+
+    for obj in raw_objects:
+        complete_object = {
+            "id": obj["_id"],
+            "score": obj["_score"],
+            "title": obj["_source"]["title"],
+        }
+
+        # DictionaryEntry
+        if obj["_index"] == ELASTICSEARCH_DICTIONARY_ENTRY_INDEX:
+            # Adding type
+            complete_object["type"] = obj["_source"]["type"]
+
+            # related audio and other required fields to be added after discussion
+
+            # Translations
+            db_object = (
+                DictionaryEntry.objects.filter(id=obj["_id"])
+                .prefetch_related("translation_set")
+                .first()
+            )
+            if db_object:
+                translation_entries = db_object.translation_set.all()
+                translations = []
+                if len(translation_entries):
+                    for translation in translation_entries:
+                        translations.append(
+                            {"id": translation.id, "text": translation.text}
+                        )
+                complete_object["translations"] = translations
+
+        complete_objects.append(complete_object)
+
+    return complete_objects
```

Comment:
I think this hydration process is going to need some work, especially once we are supporting more than one datatype. My first concern is that it would be great to not make a separate db request for every object. Second concern is that it would be great if we could reuse our existing serializers for the results instead of creating this new way of producing json.